Lib/datetime.py

```diff
@@ -1210,6 +1210,34 @@ def __reduce__(self):
         else:
             return (self.__class__, args, state)
 
+
+class IsoCalendarDate(tuple):
+
+    def __new__(cls, year, week, weekday):
+        return super().__new__(cls, (year, week, weekday))
+
+    @property
+    def year(self):
+        return self[0]
+
+    @property
+    def week(self):
+        return self[1]
+
+    @property
+    def weekday(self):
+        return self[2]
+
+    def __reduce__(self):
+        return (tuple, (tuple(self),))
+
+    def __repr__(self):
+        return (f'{self.__class__.__name__}'
+            f'(year={self[0]}, week={self[1]}, weekday={self[2]})')
```

Comment:
nitpick: try to align it with the f on the first line ;-) put it on 3 lines if needed.